### PR TITLE
Feature/users endpoint

### PR DESCRIPTION
**Pull request by mustafagonul**

### CODE REVIEW

**Comment by ardaogulcan** [comment]
test this please

File: src/models/users.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+{
+  strict: false,
+}*/
+));
+
+userSchema.plugin(slugHero, { doc: 'slugs', field: 'name' });
+
+userSchema.methods.comparePassWord = password => bcrypt.compareSync(password, this.hash_password);
+
+module.exports = mongoose.model('users', userSchema);
```
Comment (ardaogulcan) [inline src/models/users.js]: burada users'ı User olarak değiştirelim

**Review by ardaogulcan — CHANGES_REQUESTED** [review]

**Comment by mustafagonul** [comment]
- Tests are done manually by now. 
- I will add tests later.
- Collection name is changed from **users** to **User**. Cpp habits :)
- I will make a refactoring later.

@ardaogulcan

**Review by ardaogulcan — APPROVED** [review]

**Comment by ardaogulcan** [comment]
LGTM

**Pull request merged**